src/main/java/uk/gov/hmcts/darts/common/entity/CaseEntity.java

```diff
@@ -74,4 +78,11 @@ public class CaseEntity {
 
     @OneToMany(mappedBy = "courtCase")
     private List<AnnotationEntity> annotations;
+
+    @OneToMany(mappedBy = "courtCase")
```

Comment:
Do you think this relationship should have `cascade = {CascadeType.PERSIST, CascadeType.MERGE}`?